static/redfish/v1/schema/OemManager_v1.xml

```diff
@@ -26,8 +26,15 @@
                 <Annotation Term="OData.Description" String="OemManager Oem properties." />
                 <Annotation Term="OData.AutoExpand"/>
                 <Property Name="OpenBmc" Type="OemManager.OpenBmc"/>
+                <Property Name="IBM" Type="OemComputerSystem.IBM"/>
             </ComplexType>
 
+            <ComplexType Name="IBM" BaseType="Resource.OemObject">
+                <Annotation Term="OData.AdditionalProperties" Bool="true" />
+                <Annotation Term="OData.Description" String="Oem properties for IBM." />
+                <Annotation Term="OData.AutoExpand"/>
```

Comment:
Where is the property USBCodeUpdateEnabled?

Reply:
Done